apps/io-func-wallet-solution-backend/src/nonce.ts

```diff
@@ -22,3 +23,10 @@ export const insertNonce: (
   (nonce) =>
   ({ nonceRepository }) =>
     nonceRepository.insert(nonce);
+
+export const deleteNonce: (
+  nonce: string
+) => RTE.ReaderTaskEither<NonceEnvironment, Error, void> =
+  (nonce) =>
+  ({ nonceRepository }) =>
+    nonceRepository.delete(nonce);
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏ Imagine coming here in a few months and being asked: _What happens when deleting a non-existing item? Would this function fail or not?_. We both know the answer, I suggest documenting this behavior with a comment.‏